content/berks/_index.md

```diff
@@ -0,0 +1,49 @@
++++
+title = "Spotlight PA Berks County"
+link-title = "Berks County"
+title-tag = "Help us write the future of journalism in Berks County."
+description = "The future of journalism in Berks County is here. Join our historic effort to invigorate local news."
+image = "2023/09/01jf-h6pe-1sks-18dz.jpeg"
+image-description = "Berks County, PA on the horizon"
+image-credit = "Abby Drey / Centre Daily Times"
+layout = "page-landing"
+signupHed = "Join our mailing list "
+signupDek = "Sign up to receive a weekly review of articles from Spotlight PA and occasional updates about our planning project in Berks County." 
+bg-img = true
+takeover = """
+{{<featured/main-takeover
+  title = "Help us write the future of journalism in Berks County."
+  dek = "Join our historic effort to invigorate local news."
+  image = "2023/09/01jf-h6pe-1sks-18dz.jpeg"
+  signupHed = "Join our mailing list "
+  signupDek = "Sign up to receive a weekly review of articles from Spotlight PA and occasional updates about our planning project in Berks County"
+  gaCategory = "newsletters:palocal"
+  gaForm = "newsletters:palocal-full"
+  hiddenField = "newsletter/hidden-input.html"
```

Comment:
I would clone newsletter/hidden-input.html to newsletter/berks-county-hidden.html and add `<input type="hidden" name="berks_county" value="1" />` to the end of the clone. That will sign people up for Breaking News, Week In Review, and Berks County, which should be good.

Reply:
Thanks!

Updated: https://github.com/spotlightpa/poor-richard/pull/468/commits/e4927d6c5fa7942594a14e4ae2f611f7325411ec